Microsoft.Toolkit.Uwp.SampleApp/SamplePages/LottieAnimationView/LottieAnimationViewPage.xaml.cs

```diff
@@ -0,0 +1,131 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.Toolkit.Uwp.SampleApp.Models;
+using Microsoft.Toolkit.Uwp.UI.Controls;
+using Microsoft.Toolkit.Uwp.UI.Controls.Lottie;
+using Microsoft.Toolkit.Uwp.UI.Extensions;
+using Windows.Storage.Pickers;
+using Windows.UI;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+namespace Microsoft.Toolkit.Uwp.SampleApp.SamplePages
+{
+    /// <summary>
+    /// LottieAnimationViewPage sample page
+    /// </summary>
+    public sealed partial class LottieAnimationViewPage : Page, IXamlRenderListener
+    {
+        private LottieAnimationView _lottieAnimationView;
+        private RangeSelector _rangeSelector;
+        private Grid _lottieAnimationViewGrid;
+
+        public LottieAnimationViewPage()
+        {
+            InitializeComponent();
+        }
+
+        public void OnXamlRendered(FrameworkElement control)
+        {
+            Frame rootFrame = Window.Current.Content as Frame;
+            _lottieAnimationView = control.FindChildByName("Container") as LottieAnimationView;
```

Comment:
You could add a range selector in the page frame around the sample instead to use as the 'play' bar.

There's instructions in the sample app readme about that.

Reply:
How exactly is this different from the actual sample? Sorry, but I didn't understand.

Reply:
It's a bit hard to explain in text, but basically put the Range selector in the LottieAnimationViewPage.xaml instead so you can directly manipulate it vs. having to reach into the property descriptors.

See section 6 of the sample app readme.

Reply:
Oh, ok. Now I got it.
We don't have a triple range selector (for min, max **and** current frame), so I think the current implementation is just fine. You are worried only about it's positioning? I like the current implementation because I made it available to all samples, using RangeSelectorMin and RangeSelectorMax. You think I should remove that?

Reply:
You can keep the RangeSelectorMin/Max, was just suggesting an alternate to minimize some of the property code.  No worries either way.